GoedeDoelenHelpen/ClientApp/src/app/user/dashboard/progress-bar/progress-bar.component.ts

```diff
@@ -46,6 +51,30 @@ export class ProgressBarComponent implements OnInit {
 
     this.updateLevel();
     this.updateValue();
+
+    this.authService.getFB().subscribe(result => this.levelArray[1].steps[0].done = result.loggedIn);
+  }
+
+  //onClickEvents
+  stub(done: boolean) {
+    console.log("Works");
+  }
+
+  fbLoginPrompt(done: boolean): boolean {
+    if (done == false) {
+      let self: any = this;
+      return self.service.fbLoginPrompt();
```

Comment:
Is er een verschil met direct op this aanroepen?

Reply:
Direct op this kan niet, dan geeft het een compiler error, de this in dit geval is niet hetzelfde als de this in andere methods omdat dit als callbackfunctie wordt gebruikt.